dagshub/data_engine/model/datasource.py

```diff
@@ -597,14 +595,23 @@ def _upload_metadata(self, metadata_entries: List[DatapointMetadataUpdateEntry])
         # Update the status from dagshub, so we get back the new metadata columns
         self.source.get_from_dagshub()
 
-    def delete_metadata(self, path, id, key):
-        metadata_entries = [DatapointDeleteMetadataEntry(datapointId=id, url=path, key=key)]
-        self.source.client.delete_metadata(self, metadata_entries)
+    def delete_metadata_from_datapoints(self, datapoints: List[Datapoint], names: List[str]):
+        """
+        delete a metadata fields from these datapoints
+        the deleted values can be accessed using versioned query
+        with time before the deletion
+        Args:
+             datapoints - data points to act upon
+             names - name of fields to delete
```

Comment:
Same comments as on the datapoint. Make sure the docs looks nice, and run the sphinx build